pkg/test/e2e/utils/apiclient/models/metering_configuration.go

```diff
@@ -20,6 +22,14 @@ type MeteringConfiguration struct {
 	// enabled
 	Enabled bool `json:"enabled,omitempty"`
 
+	// Interval defines the number of days consulted in the metering report.
+	// +kubebuilder:default=7
+	// Minimum: 1
+	Interval int64 `json:"interval,omitempty"`
+
+	// Schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
```

Comment:
I know I'm pedantic/stupid, but without having read any other documentation, I would still be slightly confused by having both an interval and a cron schedule. Can we add a sentence like "Even if your cron schedule would create daily reports, those reports can span N number of days [though typically this is [not?] desirable.]"?

Reply:
You are right. It's not easy to decipher. I can extend description. Now that you brought this I'm thinking about renaming `Interval` to `ReportScope`. Naming things ain't easy 😅

Reply:
@xrstf I've extended the doc a bit differently than you proposed. Let me know if you find it sufficiently informative.